web/src/games/tictactoe/index.ts

```diff
@@ -26,6 +27,7 @@ export const tictactoeGameDef: IGameDef = {
   config: () => import('./config'),
   aiConfig: () => import('./ai'),
   customization: () => import('./customization'),
+  notifications: new Set([Notification.TurnChanged]),
```

Comment:
Do we really need to customize this per-game? I think as a user I would like the consistent notification behavior across all games... I.e. notify me when it is my time to play.

Also, we are not checking this for showing the menu option... Seems to be adding complexity with little gain. I think we should remove this option from IGameDef

Reply:
This is noted in the video and believe @richardaum gave a rationale -- best to check with him on the "why"

Reply:
As we have games that do not follow the turn rule, I'm not sure how notifications would be useful on those games. Or even worse - if notifications disturb players on those game non-turn based. If you guys agree, I can remove the `notifications` from game def.